radis/lbl/factory.py

```diff
@@ -193,7 +193,7 @@ class SpectrumFactory(BandFactory):
     neighbour_lines: float (:math:`cm^{-1}`)
         The calculated spectral range is increased (by ``neighbour_lines`` cm-1
         on each side) to take into account overlaps from out-of-range lines.
-        Default is ``0`` :math:`cm^{-1}`.​
+        Default is ``0`` :math:`cm^{-1}`.​ #TODO: Check weird character
```

Comment:
??

Reply:
lol it's you. See git blame below. Should I remove it?
![image](https://github.com/user-attachments/assets/1d65ef82-0294-4df8-92a0-eef0dc5e899d)

Reply:
Yes remove it :p